tests/bluetooth/tester/src/btp_gatt.c

```diff
@@ -788,6 +833,15 @@ static uint8_t alloc_value(struct bt_gatt_attr *attr, struct set_value *data)
 
 	memcpy(value->data, data->value, value->len);
 
+	/** Handle of attribute is 1 less that handle to its value */
```

Comment:
This is not true in general, there may be other descriptors before the CCCD, but I think it is ok for these tests. No change requested.